python_ta/reporters/plain_reporter.py

```diff
@@ -81,3 +82,33 @@ def _colour_messages_by_type(self, messages: Dict[str, List[NewMessage]]) -> str
                 result += self._BREAK
 
         return result
+
+    def _add_line(self, lineno: int, linetype: LineType, slice_: slice, text: str = "") -> str:
+        """Format given source code line as specified and return as str.
+
+        Called by _build_snippet, relies on _colourify.
+        """
+        snippet_so_far = super()._add_line(lineno, linetype, slice_, text)
+        if linetype == LineType.ERROR:
+            start = slice_.start or 0
+            prespace = (
+                7 + start
```

Comment:
Overall we should avoid using magic numbers like this. It looks like `PythonTaReporter` already defines one constant (`_PRE_LINE_NUM_SPACES`); please add two more constants (for the number length and the number of spaces after that). Then you can use them in `PythonTaReporter` as well as here!